src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -137,14 +136,23 @@ public function createIndex()
         );
 
         foreach ($this->locales as $locale) {
-            $localeAnalysis = clone($analysis);
-            $language = $this->analyzerLanguages[$locale]['analyzer'];
+            // Multilanguage check
+            if (count(preg_grep('/[a-z]{2}_?+[a-zA-Z]{2}/', [$locale])) > 0) {
+                $locale = strtolower($locale);
+            }
 
-            //build new index
+            // Build new index
             $index = $this->searchProvider->createIndex($this->indexName . '_' . $locale);
 
-            //create index with analysis
-            $this->setAnalysis($index, $localeAnalysis->setupLanguage($language));
+            if (array_key_exists($locale, $this->analyzerLanguages)) {
+                $localeAnalysis = clone $analysis;
+                $language = $this->analyzerLanguages[$locale]['analyzer'];
+
+                // Create index with analysis
+                $this->setAnalysis($index, $localeAnalysis->setupLanguage($language));
+            } else {
+                $index->create([]);
```

Comment:
I would prefer a very clear warning happening here that says something like,
"hey we did not crash but something is wrong here, we could not set up any analysis for this language".

Reply:
@Numkil, You mean exception?